src/surfaces/actionBar/actionBarSearch.jsx

```diff
@@ -80,12 +96,26 @@ class ActionBarSearch extends React.PureComponent {
             isMobileSearchVisible,
             onFocus,
             placeholder,
+            searchWithSelect,
             style,
             value,
         } = this.props;
+
+        const hasCorrectDataForSearchWithSelect = !_.isNil(searchWithSelect) &&
+        _.isArray(searchWithSelect.options) &&
+        _.isFunction(searchWithSelect.onChange) &&
+        (!_.isEmpty(searchWithSelect.value) || !_.isNil(searchWithSelect.value));
```

Comment:
Might be out of scope here, but can we cleanup the lodash imports?

Reply:
Fixed